### PR TITLE
Add educational example to docs

### PR DESCRIPTION
**Pull request by beorostica**
This pull request is intended to resolve issue #242

### CODE REVIEW

File: docs/make.jl [hunk 1]
```diff
@@ -29,7 +29,8 @@ end
 
 # Auxiliar function that returns an array with the subseccions of a literate seccion
 function pages_for_literate_seccion(folder)
-    return [joinpath("generated", folder, f) for f in readdir(joinpath(gen, folder)) if is_md_file(f)]
+    literate_examples = [joinpath("generated", folder, f) for f in readdir(joinpath(gen, folder)) if is_md_file(f)]
+    return push!(literate_examples, "pluto-simulation.md")
```
Comment (cncastillo) [inline docs/make.jl:33]: A more elegant solution?

Comment (beorostica) [reply docs/make.jl]: Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/248/commits/7e191c1ee9c0decfe6be83102438be868ad88e8a

File: docs/make.jl [hunk 2]
```diff
@@ -5,17 +5,23 @@ org, reps = :cncastillo, :KomaMRI
...
+exaname = "examples"
+
+# Create empty folders for in the assets directory for the literate generated sections
+mkpath(joinpath(assets, exaname))
+mkpath(joinpath(gen, exaname))
+
+# Copy the logos into the assets folder if necessary
+logo, logo_dark = joinpath(assets, "logo.svg"), joinpath(assets, "logo-dark.svg")
+(!isfile(logo)) && cp(joinpath(com, "logo.svg"), logo)
+(!isfile(logo_dark)) && cp(joinpath(com, "logo-dark.svg"), logo_dark)
```
Comment (cncastillo) [inline docs/make.jl:24]: I do not see any other repo doing this.

File: KomaMRICore/src/simulation/SimulatorCore.jl [hunk 3]
```diff
@@ -262,7 +262,7 @@ function simulate_slice_profile(
     seq::Sequence;
     z=range(-2.e-2, 2.e-2, 200), sim_params=Dict{String,Any}("Δt_rf" => 1e-6)
 )
-    sim_params["return_type"] = SpinsStateSimOutput()
```
Comment (cncastillo) [inline KomaMRICore/src/simulation/SimulatorCore.jl:265]: This should not be like this in master, please check if there is any other change

File: docs/make.jl (hunk 2 above)
Comment (beorostica) [reply docs/make.jl:24]: We aim to utilize the same icon file for both the documentation and the GUI. As Documenter searches for a logo file in its assets folder (refer to [this link](https://documenter.juliadocs.org/stable/man/guide/#Adding-a-logo-or-icon)), it is necessary to copy it there initially.

File: docs/src/pluto-simulation.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+<iframe type="text/html" src="../../../examples/pluto/simulation-example-empty.html" style="height:100vh;width:100%;"></iframe>
+```
+```@raw html
+<details><summary>View Solution</summary>
+<iframe type="text/html" src="../../../examples/pluto/simulation-example.html" style="height:100vh;width:100%;"></iframe>
+</details>
+```
+```@raw html
+<br>
+```
```
Comment (cncastillo) [inline docs/src/pluto-simulation.md:19]:
```suggestion
# MRI simulation in Pluto

This example was used in the workshop [MRI: Processing your own data](https://github.com/LIBREhub/MRI-processing-2023). In the workshop's GitHub you'll find:
 -  A presentation on [open-source software](https://github.com/LIBREhub/MRI-processing-2023/blob/main/02-simulation/day1_OpenSoftware_Nov2023.pdf) 
 - and an [introduction to MRI simulation](https://github.com/LIBREhub/MRI-processing-2023/blob/main/02-simulation/day1_MRI_simulation_Nov2023.pdf).

The participants used **KomaMRI** within a **Pluto** notebook to simulate some basic 1D sequences:  FID, Gradient Echo (GE), and Spin Echo (SE). Try to solve the problems yourself! 😼

```@raw html
<iframe type="text/html" src="../../../examples/pluto/simulation-example-empty.html" style="height:100vh;width:100%;"></iframe>
```

```@raw html
<details><summary>View Solution</summary>
<iframe type="text/html" src="../../../examples/pluto/simulation-example.html" style="height:100vh;width:100%;"></iframe>
</details>
```
```

File: docs/src/pluto-simulation.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+Have fun!
```
Comment (cncastillo) [inline docs/src/pluto-simulation.md:21]:
```suggestion
```

File: docs/make.jl (hunk 1 above)
Comment (cncastillo) [reply docs/make.jl]: Create issue to change the way the literate examples are being generated

File: docs/make.jl (hunk 2 above)
Comment (cncastillo) [reply docs/make.jl:24]: Ok

File: KomaMRICore/src/simulation/SimulatorCore.jl (hunk 3 above)
Comment (beorostica) [reply KomaMRICore/src/simulation/SimulatorCore.jl:265]: That bug should be the last of its kind. Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/248/commits/7e191c1ee9c0decfe6be83102438be868ad88e8a

File: docs/make.jl (hunk 1 above)
Comment (beorostica) [reply docs/make.jl]: Issue #251

File: docs/src/pluto-simulation.md (hunk 4 above)
Comment (beorostica) [reply docs/src/pluto-simulation.md]: Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/248/commits/ea2036a1a44166c4551226bd68e773cdfa0c4622

**Pull request merged**